### DISCUSSION TITLE
Disable specific broadcasts with debug

### DISCUSSION THREAD

**Discussion by Chikorita-Lover** [ScratchAddons/ScratchAddons#6748] [Help]
Hello! I've started using the debug extension, so I can track clone creation, broadcasts, etc.

However, I've run into a problem; I'm working with a complex project that uses tick broadcasts to run the game loop. Multiple tick broadcasts are being run each frame, meaning it is practically impossible to view helpful debug logs.

Is there anyway to disable specific broadcasts from being logged? There are still plenty of broadcasts I would like logged, just not the tick broadcasts. Thanks in advance!

**Comment by DNin01** [comment]
Wouldn't it be so cool if there was a feature where adding a comment to a broadcast block with the text "debugger-ignore"?

**Reply by BroJac5246** [reply]
> Wouldn't it be so cool if there was a feature where adding a comment to a broadcast block with the text "debugger-ignore"?

That sounds tedious.

**Reply by Chikorita-Lover** [reply]
I think a better solution could be having a list of broadcasts that are not displayed in the debugger, likely stored as a command (similar to the gamepad extension).

**Comment by WorldLanguages** [comment]
We currently do "merge" logs, but only if they're all contiguous:

![image](https://github.com/ScratchAddons/ScratchAddons/assets/17484114/f3251501-8829-476c-bbfc-386856227df6)

**Comment by DNin01** [comment]
Maybe there could be an option in an ellipsis menu next to each broadcast log where you could tell it to ignore certain broadcast messages.
